SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/TestWorker.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/ViewCoordinator.h"
+#include "TestConnectionHandler.h"
+
+namespace SpatialGDK
+{
+class FTestWorker
+{
+public:
+	explicit FTestWorker(TArray<Worker_ComponentSetId> ComponentSetIds, FSendMessageCallback InSendMessageCallback = FSendMessageCallback(),
```

Comment:
ComponentSetIds should be const ref?

Reply:
It's a sink so it gets moved in by value. As it will be immediately moved from and likely often passed in by rvalue it's likely this will also be faster.

Reply:
Although speed wasn't actually a concern here. The main point is that it owns what gets passed in, so the default is by value.